charters/DataOperations/charter.md

```diff
@@ -0,0 +1,101 @@
+# [Data Operations](mailto:data-operations-team@data.humancellatlas.org)
+
+
+## Description
+
+The Data Coordination Platform (DCP) Data Operations Team (DataOps) is a user-focused group aiming to connect the scientific community to the DCP through the effective tools and resources generated by the DCP which maximize the value and use of HCA data. This involves identifying the value added for the HCA consumers from various DCP components and the needs and expectations of the consumers, and connecting these communities through useful, fully vetted products and their distribution. These products include data, metadata, analysis tools, data access features. DataOps ensures that data flows through the DCP end-to-end in an efficient manner, is appropriately validated, and fulfills the needs and expectations of data consumers. DataOps maintains a DCP-wide vision through their internal focus on end-to-end data flow, as well as their community-focused goal of conveying the value of the entire DCP.
+
+## Definitions
+
+*Data contribution*: a set of raw data derived from the same experimental process, and the metadata describing that process
+
+*Release* (verb): to make metadata and data openly accessible
+
+*Data distribution*: the packaging of metadata and data from multiple data contributions in meaningful ways, along with relevant analysis pipelines, possible integrative analysis results, user vignettes, access features, and any announcements or documentation supporting it
+
+*HCA Snapshot*: a type of data distribution that occurs on a regular cadence containing all HCA data currently available that meets predetermined criteria, such as quality standards
```

Comment:
When speaking of quality, it would be helpful to know if you mean quality assurance, quality control or both.

Reply:
Both

Reply:
@TimothyTickle does this answer your questions, or is there still more to resolve here?